pkg/controller/node/node_controller.go

```diff
@@ -368,19 +370,22 @@ func isWindows(node *corev1.Node) bool {
 func (ctrl *Controller) reconcileMaster(node *corev1.Node) {
 	mastersSchedulable, err := ctrl.getMastersSchedulable()
 	if err != nil {
-		goerrs.Wrap(err, "Getting scheduler config failed")
+		err = fmt.Errorf("getting scheduler config failed: %w", err)
+		glog.Error(err)
```

Comment:
Out of curiosity, why did you add these `glog.Error(err)` statements?

Reply:
@cheesesashimi I think this relates to the discussion here: https://github.com/openshift/machine-config-operator/pull/2868#discussion_r770073515

Went with option 1.

Reply:
Ahhh yes, that's right. Sorry!